### PR TITLE
Stop search engines from crawling the tool

### PR DESCRIPTION
**Pull request by brenetic**
We do not want any results from this tool appearing in search engines results.

https://trello.com/c/6qUAhPQi/304-stop-internal-apps-and-deprecated-stuff-appearing-in-search-engines

### CODE REVIEW

**Comment by suzannehamilton** [comment]
I'm not very familiar with Flask, but does adding robots.txt to the project root mean that it will be served on the site root? The other files in the project root aren't part of the site itself.

**Comment by brenetic** [comment]
@suzannehamilton thanks! updated

File: explorer/api.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ def title_or_slug(item):
     return item.get("slug", "")
 
 
+@app.route("/robots.txt")
```
Comment (MatMoore) [inline explorer/api.py:22]: this is a decorator for the `main` function, so I'd expect it to still serve the compare page/main page at `/robots.txt`.

Should this use something like this? http://flask.pocoo.org/docs/0.12/api/#flask.send_from_directory

I'm assuming there's not a simple way to make heroku serve this?

**Review by MatMoore — CHANGES_REQUESTED** [review]
see inline comment

**Review by MatMoore — APPROVED** [review]

File: explorer/api.py [hunk 2]
```diff
@@ -19,6 +19,10 @@ def title_or_slug(item):
...
+def robots():
+    return send_static_file("robots.txt")
```
Comment (suzannehamilton) [inline explorer/api.py]: Does robots.txt have to be in the static directory for this to work?

**Review by MatMoore — APPROVED** [review]

**Pull request merged**